server/tsconfig.json

```diff
@@ -2,12 +2,24 @@
 	"compilerOptions": {
 		"target": "es6",
 		"module": "commonjs",
+		"moduleResolution": "node",
+		"noImplicitAny": false,
+		"resolveJsonModule": true,
 		"outDir": "./dist",
+		"baseUrl": ".",
 		"rootDir": "./src",
 		"strict": true,
 		"esModuleInterop": true,
+		"experimentalDecorators": true,
+		"emitDecoratorMetadata": true,
```

Comment:
Necessary for [sequelize-typescript](https://github.com/sequelize/sequelize-typescript?tab=readme-ov-file#installation)